cmd/protoc-gen-go-grpc/README.md

```diff
@@ -14,7 +14,8 @@ To restore this behavior, set the option `require_unimplemented_servers=false`.
 E.g.:
 
 ```sh
-  protoc --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false[,other options...] \
+   protoc --go-grpc_out=. --go-grpc_opt=require_unimplemented_servers=false \
+          --go-grpc_opt=[other options...]
```

Comment:
need \ at the end and everything can be in single line

Reply:
done please check